rpi-eeprom-update

```diff
@@ -192,12 +222,20 @@ checkDependencies() {
       die "Bootloader updates directory ${FIRMWARE_IMAGE_DIR} not found."
    fi
 
+   if ! vl805 -h > /dev/null 2>&1; then
+      die "vl805 command not found"
+   fi 
+
    if vcgencmd bootloader_config | grep -qi "Command not registered"; then
-      die "vcgencmd: 'bootloader_config' command not supported. Please update VC firmware"
+      die "vcgencmd: 'bootloader_config' command not supported. Please update VC firmware and reboot."
+   fi
+
+   if ! sha256sum --version > /dev/null 2>&1; then
```

Comment:
Unfortunately busybox (which implements `sha256sum`) returns exit code=1 for `sha256sum --version`, so this test fails even when a working `sha256sum` is available.

Would it be possible to use instead:
```
if ! command -v sha256sum >/dev/null`; then
```
and the same for `vl805` and `flashrom`, as I think that might be more reliable.

If not `command`, then `which` should always be available:
```
  if ! which sha256sum >/dev/null; then
```

(From what I've read, `command` tends to be preferred to `which` these days).